File: Python Basics/02_Conditional_Statements/task_4.py
number = float(input())
measure_unit_input = input()
measure_unit_output = input()

if measure_unit_input == 'm' and measure_unit_output == 'cm':
    print(f"{number * 100:.3f}")
elif measure_unit_input == 'm' and measure_unit_output == 'mm':
    print(f"{number * 1000:.3f}")

elif measure_unit_input == 'cm' and measure_unit_output == 'mm':
    print(f"{number * 10:.3f}")
elif measure_unit_input == 'cm' and measure_unit_output == 'm':
    print(f"{number * 0.01:.3f}")

elif measure_unit_input == 'mm' and measure_unit_output == 'cm':
    print(f"{number * 0.1:.3f}")
elif measure_unit_input == 'mm' and measure_unit_output == 'm':
    print(f"{number * 0.001:.3f}")

File: JS Advanced/Exercises Objects & Composition/task4.js
function compose(input) {
    let result = []

    for (const iterator of input) {
        let [hero, level, items] = iterator.split(' / ');
        level = Number(level);
        items = items ? items.split(', ') : [];

        current_obj = {
            name: hero,
            level: level,
            items: items,
        };

        result.push(current_obj)
    }
      
    console.log(JSON.stringify(result))
}

compose(['Isacc / 25 / Apple, GravityGun',
    'Derek / 12 / BarrelVest, DestructionSword',
    'Hes / 1 / Desolator, Sentinel, Antara']
)